extensions/cornerstone/src/commandsModule.ts

```diff
@@ -397,6 +395,26 @@ function commandsModule({
       viewports.forEach((_, index) => cineService.setCine({ id: index, isPlaying: false }));
     },
 
+    navigateToMeasurementItems: props => {
```

Comment:
Could you provide the documentation, please? I'm not really sure what `measurementItems` are. Also, what does `navigate` mean in this context? Does it mean to jump to something?